xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -602,9 +602,11 @@ void CGUIWindowPictures::OnItemInfo(int itemNumber)
 
 std::string CGUIWindowPictures::GetStartFolder(const std::string &dir)
 {
-  std::string lower(dir); StringUtils::ToLower(lower);
-  if (lower == "plugins" || lower == "addons")
+  if (StringUtils::EqualsNoCase(dir, "plugins") ||
```

Comment:
Doesn't the `StringUtils::ToLower(lower)` have the effect of a case-insensitive comparison? So this isn't a change in functionality, is it?